app.go

```diff
@@ -52,10 +52,6 @@ type Option interface {
 	apply(*App)
```

Comment:
you might want to specify `fmt.Stringer` so an option can't get added without it

Reply:
I intentionally kept that out. Per the commit message,

> Note that we've intentionally kept fmt.Stringer out of the fx.Option
> interface: We don't want to commit to having options printable but it's
> nicer if they are.

Reply:
If downstream someone wants to print all the options that were used in the creation of the app, I don't really see a problem with that (I do see a problem of some of them being string-able and some not).

It seems like somewhat arbitrary line, why can't we commit to requiring them all to have a `String()`?

Reply:
I was envisioning arguments that could not reliably be turned into strings but I suppose everything can support a placeholder. I'll add it to the interface.